src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -206,25 +201,18 @@ const SignOrExecuteForm = ({
 
         {tx && <DecodedTx tx={tx} txId={txId} />}
 
-        {canExecute && !onlyExecute && (
-          <FormControlLabel
-            control={<Checkbox checked={shouldExecute} onChange={onExecuteTxCheckbox} />}
-            label="Execute transaction"
-            sx={{ mb: 1 }}
-          />
-        )}
-
-        {tx && (
```

Comment:
@iamacook I'm removing this check, will try to move it to the threshold component itself somehow.

Reply:
Ok, please mention your changes in the [relevant issue](https://www.notion.so/gnosis-safe/7e47da8dd78b4027abdd05418c6aeb0e?v=f0b305af71664ad7a0ab79bd3d7addb5&p=9f5ecc0913ac4626ae9994b06fa41e01&pm=s) when you are done.